subprojects/common/report-api/src/main/kotlin/com/avito/report/model/IncidentElement.kt

```diff
@@ -13,5 +15,5 @@ public data class IncidentElement(
     val type: String? = null,
     val origin: String? = null,
     val className: String? = null,
-    val data: String? = null
+    val data: JsonElement? = null
 )
```

Comment:
Let's mark json metadata. RM client parses this model from json.

And what about `com.avito.report.internal.model.IncidentElement` from report-viewer? Are they the same?